src/Context/QuizProvider.tsx

```diff
@@ -0,0 +1,17 @@
+import { createContext, useContext, useReducer } from "react";
+import { initialState, quizReducer } from "../Reducer/quizReducer";
```

Comment:
I think that you can define the initial state in this file itself, to increase ease of understanding.

Reply:
However, I have used the initialState to on the RESET action. Hence it is kept on the reducer file.